node/lib/TranslatorInfo.ts

```diff
@@ -0,0 +1,26 @@
+
+/**
+ * Information about a translator, as loaded from the translator's manifest.
+ */
+export class TranslatorInfo {
+    /**
+     * Fully-qualified name of the translator (required).
+     */
+    public readonly name: string;
```

Comment:
The translator manifest doesn't currently have anything like this, but I guess it makes sense to add it now.

---
In reply to: [71904642](https://github.com/opent2t/opent2t/pull/1#discussion_r71904642) [](ancestors = 71904642)